tests/Aspire.Hosting.Tests/Dashboard/DashboardLifecycleHookTests.cs

```diff
@@ -523,7 +523,7 @@ private static DashboardEventHandlers CreateHook(
             new TestHostApplicationLifetime(),
             new Hosting.Eventing.DistributedApplicationEventing(),
             rewriter,
-            new FileSystemService()
+            new FileSystemService(new ConfigurationBuilder().Build())
```

Comment:
nit: you could use the `IConfiguration` instance passed in here for configurability.